src/Application/Actions/Donations/Update.php

```diff
@@ -45,7 +44,7 @@ public function __construct(
         private DonationRepository $donationRepository,
         private EntityManagerInterface $entityManager,
         private SerializerInterface $serializer,
-        private StripeClient $stripeClient,
+        private Stripe $stripe,
```

Comment:
This is now our abstraction of Stripe that is simpler to use than the off the shelf library as well as being sometimes real and sometimes stub depending what we need that day.